swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
+from .item import Item
+
+class Clothing(Item):
+        
+        def __init__(self, condition=None):
+            if condition == None:
+                super().__init__(category = "Clothing")
```

Comment:
The style for passing named params or setting default values typically leaves out the spaces around the `=`. ([PEP8](https://www.python.org/dev/peps/pep-0008/#:~:text=Don't%20use%20spaces%20around%20the%20=%20sign))
```suggestion
                super().__init__(category="Clothing")
```